L/LibTeXPrintf/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder, Pkg
 
 name = "LibTeXPrintf"
-version = v"1.11.0"
+version = v"1.14.0"
```

Comment:
Source didn't change?

Reply:
No, but they tagged a [new release](https://github.com/bartp5/libtexprintf/releases/tag/v1.14) to the current main. The answer to "why do you not set the source to the release?" is that in the release file, the `autogen.sh` file is not present, and the patch for `configure.ac` does not apply because they have already run it.